i6_models/parts/e_branchformer/merge.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+__all__ = ["MergerV1Config", "MergerV1"]
+
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class MergerV1Config(ModelConfiguration):
+    """
+    Attributes:
+        input_dim: input dimension
+        kernel_size: kernel size of the depthwise convolution layer
+        dropout: dropout probability
+    """
+
+    input_dim: int
+    kernel_size: int
+    dropout: float
+
+
+class MergerV1(nn.Module):
+    def __init__(self, model_cfg: MergerV1Config):
+        """
+        The merge module to merge the outputs of local extractor and global extractor
+        Here we take the best variant from the E-branchformer paper, refer to
+        https://arxiv.org/abs/2210.00077 for more merge module variants
+        """
+        super().__init__()
+
+        self.depthwise_conv = nn.Conv1d(
+            in_channels=model_cfg.input_dim * 2,
+            out_channels=model_cfg.input_dim * 2,
+            kernel_size=model_cfg.kernel_size,
+            padding=(model_cfg.kernel_size - 1) // 2,
+            groups=model_cfg.input_dim * 2 // 2,
```

Comment:
why is here `groups` not the same as `in_channels`?

Reply:
Yes this should be same as `in_channels`. See also here for reference: https://github.com/espnet/espnet/blob/master/espnet2/asr/encoder/e_branchformer_encoder.py#L102C32-L102C32